notebooks/lesson05_random-forest-deep-dive.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>The for loop over a list of different object is nice and compact but maybe requires a hint? I remember last year I did a very similar thing to show them overfitting etc. and the for loops were not their friends</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
<p>thanks!</p>